### PR TITLE
Added instructions how to deploy CVAT behind a proxy

### PR DESCRIPTION
**Pull request by azhavoro**
Added instructions how to deploy CVAT behind a proxy

### Motivation and context
Some users have experienced issues when deploying CVAT with serverless functions behind a proxy. This PR should simplify the process.

### How has this been tested?
Manually

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: components/analytics/docker-compose.analytics.yml [hunk 1]
```diff
@@ -63,7 +63,7 @@ services:
       DJANGO_LOG_SERVER_PORT: 5000
       DJANGO_LOG_VIEWER_HOST: kibana
       DJANGO_LOG_VIEWER_PORT: 5601
-      no_proxy: kibana,logstash,${no_proxy}
+      no_proxy: kibana,logstash,nuclio,${no_proxy}
```
Comment (nmanovic) [inline components/analytics/docker-compose.analytics.yml:66]: why do we need it here?

File: cvat/apps/documentation/installation.md [hunk 2]
```diff
@@ -245,6 +246,28 @@ server. Proxy is an advanced topic and it is not covered by the guide.
 
 ## Advanced topics
 
+### Deploying CVAT behind a proxy
+If you deploy CVAT behind a proxy and do not plan to use any of [serverless functions](#semi-automatic-and-automatic-annotation)
+for automatic annotation, the exported enviroment variables
+`http_proxy`, `https_proxy` and `no_proxy` should be enuagh to build images.
```
Comment (nmanovic) [inline cvat/apps/documentation/installation.md:252]: Could you please check words like 'enuagh'?

File: cvat/apps/documentation/installation.md [hunk 3]
```diff
@@ -245,6 +246,28 @@ server. Proxy is an advanced topic and it is not covered by the guide.
...
+Otherwise please create or edit the file `~/.docker/config.json` in the home directory of the user
```
Comment (nmanovic) [inline cvat/apps/documentation/installation.md:253]: cool feature 👍

File: components/analytics/docker-compose.analytics.yml (hunk 1 above)
Comment (azhavoro) [reply components/analytics/docker-compose.analytics.yml:66]: Without this change, if we create images with the analytic component, no_proxy will be overridden by this value without `nuclio`.

Comment (nmanovic) [reply components/analytics/docker-compose.analytics.yml:66]: OK. I see. I thought the change was necessary for analytics itself.

**Pull request merged**